### ISSUE TITLE
Overwriting dataset files instead of deletion.

### ISSUE THREAD

**Issue by tanishqvyas** [tanishqvyas/datascienceProject#3]
@shreyapr6 @Raksha-Ramesh 
In the current implementation, the nan folder is first deleted and then a new folder is made. We should find a way to overwrite existing content.

**Comment by Raksha-Ramesh** [comment]
It's deleting because we're using shutil rmtree, right?
We have to find an alternative to that

**Comment by tanishqvyas** [comment]
@Raksha-Ramesh Yes, exactly. We need to find just one function which simply overwrites if possible.
